### PR TITLE
PIM-5176: Customisation of columns not saved in "Column Selection"

### PR DESCRIPTION
**Pull request by grena**
| Q                 | A
| ----------------- | ---
| Specs             | Y
| Behats            | N
| Blue CI           | Y
| Changelog updated | Y
| Review and 2 GTM  | Y

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Resources/config/requirejs.yml [hunk 1]
```diff
@@ -96,3 +96,5 @@ config:
         oro/datafilter/filters-manager:             pimdatagrid/js/datafilter/filters-manager
         oro/datafilter/collection-filters-manager:  pimdatagrid/js/datafilter/collection-filters-manager
         oro/datafilter-builder:                     pimdatagrid/js/datafilter-builder
+
+        pim/template/datagrid/configure-columns-action: pimdatagrid/templates/configure-columns-action.html
```
Comment (solivier) [inline src/Pim/Bundle/DataGridBundle/Resources/config/requirejs.yml]: missing new end line

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/action/configure-columns-action.js [hunk 2]
```diff
@@ -270,15 +237,28 @@ define(
...
+                            code: displayedCode,
+                            position: index
+                        }
+                    });
+
                     var columnList = new ColumnList();
                     _.each(columns, function(column) {
-                        column.displayed = _.indexOf(displayedCodes, column.code) !== -1;
+                        var displayedCode = _.findWhere(displayedCodes, {code: column.code});
+                        if (displayedCode) {
```
Comment (phaseinducer) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/action/configure-columns-action.js]: `!_.isUndefined()` ?

**Comment by phaseinducer** [comment]
Small comment then GTM

File: src/Pim/Bundle/DataGridBundle/Resources/config/requirejs.yml (hunk 1 above)
Comment (grena) [inline src/Pim/Bundle/DataGridBundle/Resources/config/requirejs.yml]: done

**Comment by solivier** [comment]
GTM

**Pull request merged**